about.html

```diff
@@ -2,13 +2,28 @@
 <html>
 <head>
   <title>Abraham Kwak - about</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-  <nav>
-    <a href="index.html">Home</a>
-    <a href="about.html">About</a>
-  </nav>
-  <h2>My biography:</h2>
-  <p>Born and raised in NJ/NYC area, I began working in the digital media industry 6 years ago. I started as a junior web developer but my career path led me towards user experience design. Today I am working hard to become a software engineer and fulfill my goal to become a designer/engineer hybrid.</p>
+  <div class="main-container">
+    <div class="head-section">
+      <h1><a href="index.html">Abraham Kwak</a></h1>
+      <h3>Product Designer and Full Stack Developer</h3>
+      <nav>
+        <ul id="nav">
```

Comment:
I love the `ul` inside the `nav` to bring order to the elements inside, but when you're naming your ids it's useful to think "am I going to be able to tell the difference between `nav` and `#nav` at a glance, or is my CSS gonna get _confusing_?"